### ISSUE TITLE
Not using softmax with cross entropy loss in 001b_fit.ipynb

### ISSUE THREAD

**Issue by aakashns** [fastai/fastai_v1#9]
https://github.com/fastai/fastai_v1/blob/ec6fc340ea9d6077f04ea87e2971d5ad8c841f4a/dev_nb/001b_fit.ipynb#L273

In the section "Simplify nn.Sequential layers", the loss `F.cross_entropy` is applied directly to the flattened output, without a softmax layer. Shouldn't it be one of the following:
* Apply softmax to the outputs and use cross entropy loss
* Apply log softmax to the outputs and use NLL loss

P.S.: Thanks for the step-by-step explanation!

**Comment by bny6613** [comment]
In pytorch cross_entropy loss already has softmax in it - `cross_entropy = log_softmax + nll_loss`. https://pytorch.org/docs/stable/nn.html#torch.nn.functional.cross_entropy

**Comment by aakashns** [comment]
Great, thanks for the clarification!
